sprockets/mixins/correlation/__init__.py

```diff
@@ -7,5 +7,5 @@ def __init__(self, *args, **kwargs):
             raise ImportError
 
 
-version_info = (2, 0, 1)
+version_info = (2, 0, 2)
```

Comment:
This is a breaking change and should be 3.0.

Reply:
Fair point. Increased version to 3.0.